oadp-1.5: Update Konflux references

rayfordj opened the pull request:
Update konflux-ci image references

rayfordj committed:
oadp-1.5: Update Konflux references

Update konflux-ci image references

Changes committed via automation for oadp-1-5/oadp-non-admin.

oadp-snyk commented:
![](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/pr-banner-default.svg)

### :tada: **Snyk checks have passed. No issues have been found so far.**

:white_check_mark: **security/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/hybrid-platforms-openshift-api-for-data-protection/pr-checks/f8e5b819-b92f-41cc-83ea-a95ade8079fc?refs=5e24cbaf64c8985497f163466d3ce9e34a329a03&source=prChecksComment)

:white_check_mark: **license/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/hybrid-platforms-openshift-api-for-data-protection/pr-checks/f8e5b819-b92f-41cc-83ea-a95ade8079fc/license?refs=5e24cbaf64c8985497f163466d3ce9e34a329a03&source=prChecksComment)

rayfordj committed:
Update apply-tags

https://github.com/migtools/oadp-non-admin/pull/304/checks?check_run_id=47373306388

```
invalid input params for task apply-tags: missing values for these
params which have no default values: [IMAGE_URL IMAGE_DIGEST]
```
